sdk/servicebus/azure-servicebus/azure/servicebus/common/utils.py

```diff
@@ -23,13 +23,22 @@
 
 def get_running_loop():
     try:
-        import asyncio
+        import asyncio  # pylint: disable=import-error
         return asyncio.get_running_loop()
     except AttributeError:  # 3.5 / 3.6
-        loop = asyncio._get_running_loop()  # pylint: disable=protected-access
+        loop = None
+        try:
+            loop = asyncio._get_running_loop()  # pylint: disable=protected-access
+        except AttributeError:
+            logger.warning('This version of Python is deprecated, please upgrade to >= v3.5.3')
         if loop is None:
-            raise RuntimeError('No running event loop')
+            logger.warning('No running event loop')
+            loop = asyncio.get_event_loop()
         return loop
+    except RuntimeError:
+        # For backwards compatibility, create new event loop
+        logger.warning('No running event loop')
+        return asyncio.get_event_loop()
```

Comment:
Is this for calling get_running_loop( ) in a sync method?

Reply:
My understanding was that the additional exception blocks were to add additional backcompat robustness motivated by issues on the EH side of the world.  Unfortunately I don't have any additional context on the specific impetus, we can follow up with Anna if we think there's a risk.